client-service/src/main/java/com/telecominfraproject/wlan/client/controller/ClientController.java

```diff
@@ -23,6 +23,8 @@
 import com.telecominfraproject.wlan.client.models.events.ClientSessionChangedEvent;
 import com.telecominfraproject.wlan.client.models.events.ClientSessionRemovedEvent;
 import com.telecominfraproject.wlan.client.session.models.ClientSession;
+import com.telecominfraproject.wlan.client.validator.ClientSessionValidator;
```

Comment:
I see that with these changes we will not be able to create **client sessions** with SSIDs that are too long, but I do not think the root of the problem is addressed - it is still possible to provision SSIDs with long names using /portal/profile APIs. The validation should happen in com.telecominfraproject.wlan.profile.ssid.models.SsidConfiguration class, everything else is secondary.